server/controllers/user.js

```diff
@@ -11,9 +15,28 @@ const UsersController = {
         if (err) {
           throw err;
         }
+    res.redirect('/:username')
       });
     }
+  },
+
+  userProfile: async (req, res) => {
```

Comment:
This userProfile function aims to show userwork out data.  At the moment, I will show the user data and try to connect to the front end and populate the data first.  Will amend it to show workout and exercise later.